src/test/java/org/folio/rest/support/builders/RequestRequestBuilder.java

```diff
@@ -718,7 +750,33 @@ public RequestRequestBuilder withPickupServicePointId(UUID pickupServicePointId)
       this.cancellationAdditionalInformation,
       this.cancelledDate,
       this.position,
-      pickupServicePointId);
+      pickupServicePointId,
+      tags);
+  }
+
+  public RequestRequestBuilder withTags(Tags tags) {
+    return new RequestRequestBuilder(
+      this.id,
+      this.requestType,
+      this.requestDate,
+      this.itemId,
+      this.requesterId,
+      this.proxyId,
+      this.fulfilmentPreference,
+      this.deliveryAddressTypeId,
+      this.requestExpirationDate,
+      this.holdShelfExpirationDate,
+      this.itemSummary,
+      this.requesterSummary,
+      this.proxySummary,
+      this.status,
+      this.cancellationReasonId,
+      this.cancelledByUserId,
+      this.cancellationAdditionalInformation,
+      this.cancelledDate,
+      this.position,
+      pickupServicePointId,
```

Comment:
As mentioned above, to make this consistent, change `pickupServicePointId` to `this.pickupServicePointId`.